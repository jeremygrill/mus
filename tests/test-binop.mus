Int main(){

	Int a;
	Int b;
	Int c;
	Int d;
	
	a = 5;
	b = 10;
	c = a * b;  //c = 50
	
	print(c);
	
	b = c * b; //b = 500
	
	print(b);

	c = c / 25; //c = 2
	
	print(c);

	c = c + 10 - 1 * 3;
	
	print(c);

	return 0;

}
