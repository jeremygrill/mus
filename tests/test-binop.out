50
500
2
9
